### PR TITLE
T1055 svchost writing a file to a unc path

### PR DESCRIPTION
**Pull request by apbeers**
**Details:**
svchost.exe writing a non microsoft office file to a file with a UNC path
This works by copying cmd.exe to a file and naming in svchost.exe, then copying a file to the localhost\c$ folder

**Testing:**
tested on windows 10 vm

**Associated Issues:**
no issues fixed

### CODE REVIEW

**Comment by ForensicITGuy** [comment]
Hi @apbeers, quick question- Did you mean for this to be under Masquerading rather than Process Injection? I'm not seeing the injection use case for this test (although I could be missing it).

**Comment by apbeers** [comment]
@ForensicITGuy I was kinda spitballing on which technique it was, the goal is to detect a masquerading process trying to spread to a network share. so maybe there's a better technique for it?

**Comment by ForensicITGuy** [comment]
@apbeers No problem, this should probably fit well under the Masquerading section (T1036).

File: atomics/T1055/T1055.yaml [hunk 1]
```diff
@@ -102,3 +102,21 @@ atomic_tests:
     name: command_prompt
     command: |
       .\bin\#{exe_binary}
+
+- name: svchost writing a file to a UNC path
+  description: | 
+    svchost.exe writing a non microsoft office file to a file with a UNC path
```
Comment (keithmccammon) [inline atomics/T1055/T1055.yaml]:
```suggestion
    svchost.exe writing a non-Microsoft Office file to a file with a UNC path.
```

File: atomics/T1055/T1055.yaml [hunk 2]
```diff
@@ -102,3 +102,21 @@ atomic_tests:
...
+    This works by copying cmd.exe to a file and naming in svchost.exe, then copying a file to the localhost\c$ folder
```
Comment (keithmccammon) [inline atomics/T1055/T1055.yaml]:
```suggestion
    This works by copying cmd.exe to a file, naming it svchost.exe, then copying a file to the localhost\c$ folder.
```

**Pull request merged**